src/JustSaying/Messaging/Middleware/Metrics/HandlerMiddlewareBuilderExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using JustSaying.Messaging.Middleware.Handle;
+using JustSaying.Messaging.Monitoring;
+
+namespace JustSaying.Messaging.Middleware.Metrics
```

Comment:
Similar comments about discoverability putting everything in discrete namespaces.

Reply:
These are all now in the same namespace, `JustSaying.Messaging.Middleware`, and I've marked the folders as not contributing to namespaces to avoid the squiggly in jetbrains products.